### PR TITLE
Remove unused data after removing qchem dependency

### PR DESCRIPTION
**Pull request by soranjh**
This PR removes unused data in `_vqe_defaults` function and `VQE_heavy` class. These data were used by `qchem` to generate the UCCSD ansatz for hydrogen and lithium hydride. After removing the qchem dependency in the benchmark, the data are not needed anymore.

### CODE REVIEW

**Review by albi3ro — APPROVED** [review]
Thanks for fixing this up :)

File: benchmarks/benchmark_functions/default_settings.py [hunk 1]
```diff
@@ -126,7 +116,7 @@ def _qaoa_defaults(hyperparams):
 
     # if device name is given, create device
     if isinstance(device, str):
-        device = qml.device(device, wires=len(graph.nodes), analytic=False)
+        device = qml.device(device, wires=len(graph.nodes), shots=None)
```
Comment (mariaschuld) [inline benchmarks/benchmark_functions/default_settings.py:119]: Good catch!

File: benchmarks/benchmark_functions/default_settings.py [hunk 2]
```diff
@@ -99,11 +91,9 @@ def _vqe_defaults(hyperparams):
 
     options_dict = {"interface": interface, "diff_method": diff_method, "optimize": optimize}
 
+    # if device name is given, create device
     if isinstance(device, str):
-        if device == "cirq.pasqal":
-            device = qml.device(device, wires=n_wires, control_radius=1.5)
-        else:
-            device = qml.device(device, wires=n_wires)
+        device = qml.device(device, wires=len(hf_state), shots=None)
```
Comment (mariaschuld) [inline benchmarks/benchmark_functions/default_settings.py:96]: Would that not mean we cannot run on the pasqal device (at least not when passing a string)?

**Review by mariaschuld — APPROVED** [review]

File: benchmarks/benchmark_functions/default_settings.py (hunk 2 above)
Comment (albi3ro) [reply benchmarks/benchmark_functions/default_settings.py:96]: Given this is not part of the device benchmark suite, it'll probably be fine.

**Pull request merged**